cluster/src/dunit/scala/io/snappydata/cluster/SecureQueryRoutingDUnitTest.scala

```diff
@@ -0,0 +1,246 @@
+  }
+
+  def insertRows1(numRows: Int, serverHostPort: Int, tableName: String,
+      user: String, pass: String): Unit = {
+    val conn = netConnection(serverHostPort, user, pass)
+    // scalastyle:off println
+    println(s"insertRows1: Connected to $serverHostPort")
+    // scalastyle:on println
+
+    val stmt1 = conn.createStatement()
+    val rows = (1 to numRows).toSeq
+    try {
+      var i = 1
+      rows.foreach(d => {
+        stmt1.addBatch(s"insert into $tableName values($i, $i, '$i')")
+        i += 1
+        if (i % 1000 == 0) {
+          stmt1.executeBatch()
+          i = 0
+        }
+      })
+      stmt1.executeBatch()
+
+      // scalastyle:off println
+      println(s"insertRows1: committed $numRows rows")
+      // scalastyle:on println
+    } finally {
+      stmt1.close()
+      conn.close()
+    }
+  }
+
+  def insertRows2(numRows: Int, serverHostPort: Int, tableName: String,
+      user: String, pass: String): Unit = {
+    val conn = netConnection(serverHostPort, user, pass)
+    // scalastyle:off println
+    println(s"insertRows2: Connected to $serverHostPort")
+    // scalastyle:on println
+
+    val stmt1 = conn.createStatement()
+    val rows = (1 to numRows).toSeq
+    try {
+      rows.foreach(i => {
+        stmt1.executeUpdate(s"insert into $tableName values($i, $i, '$i')")
+      })
+
+      // scalastyle:off println
+      println(s"insertRows2: committed $numRows rows")
+      // scalastyle:on println
+    } finally {
+      stmt1.close()
+      conn.close()
+    }
+  }
+
+  def verifyQuery1(qryTest: String, stmt_rs: ResultSet): Unit = {
+    val builder = StringBuilder.newBuilder
+
+    var index = 0
+    while (stmt_rs.next()) {
+      index += 1
+      val stmt_i = stmt_rs.getInt(1)
+      val stmt_j = stmt_rs.getInt(2)
+      val stmt_s = stmt_rs.getString(3)
+      if (index % 1000 == 0) {
+        builder.append(s"$qryTest Stmt: row($index) $stmt_i $stmt_j $stmt_s ").append("\n")
+      }
+    }
+    builder.append(s"$qryTest Stmt: Total number of rows = $index").append("\n")
+    // scalastyle:off println
+    println(builder.toString())
+    // scalastyle:on println
+    assert(index == 40000)
+  }
+
+  def query1(serverHostPort: Int, tableName: String, user: String, pass: String): Unit = {
+    val conn = netConnection(serverHostPort, user, pass)
+    // scalastyle:off println
+    println(s"query1: Connected to $serverHostPort")
+    // scalastyle:off println
+
+    val stmt1 = conn.createStatement()
+    try {
+      val qry1 = s"select ol_int_id, ol_int2_id, ol_str_id " +
+          s" from $tableName " +
+          s" where ol_int_id < 5000000 " +
+          s""
+      val rs1 = stmt1.executeQuery(qry1)
+      verifyQuery1(qry1, rs1)
+      rs1.close()
+      // Thread.sleep(1000000)
+    } finally {
+      stmt1.close()
+      conn.close()
+    }
+  }
+
+  def createTable1(serverHostPort: Int, tableName: String, user: String, pass: String): Unit = {
```

Comment:
Instead of names like 1 and 2 appended to method names we should use a little informative names otherwise you need to look at the code to figure out the difference between 1 and 2 etc.